modules/ocha_ai_chat/components/chat-form/chat-form.js

```diff
@@ -175,7 +187,73 @@
       targetElements.forEach((el) => {
         observer.observe(el, config);
       });
-    }
+    },
+
+    /**
+     * Copy to Clipboard
+     *
+     * Needs to be run every time the form reloads. It will find all the copy
+     * buttons and attach an event listener that copies individual answers to
+     * the user's clipboard.
+     *
+     * Adapted from CD Social Links in CD v9.4.0
+     *
+     * @see https://github.com/UN-OCHA/common_design/blob/v9.4.0/libraries/cd-social-links/cd-social-links.js
+     */
+    copyToClipboard: function () {
+      // Collect all "copy" URL buttons.
+      const copyButtons = document.querySelectorAll('.feedback-button--copy');
+
+      // Process links so they copy URL to clipboard.
+      copyButtons.forEach(function (el) {
+        // First, define the status element for each button.
+        var status = el.parentNode.querySelector('[role=status]');
+
+        // Add our event listener so people can copy to clipboard.
+        el.addEventListener('click', function (ev) {
+          var tempInput = document.createElement('input');
+          var textToCopy = document.querySelector('#' + el.dataset.for).innerText;
```

Comment:
```suggestion
          var textToCopy = document.querySelector('#' + el.dataset.for).textContent;
```

`innerText` is empty in Safari (17.1) while `textContent` seems to work fine in all the browsers I tried with (Chrome, Firefox, Safari, Edge).